src/libs/antares/study/parameters.cpp

```diff
@@ -548,6 +549,8 @@ static bool SGDIntLoadFamily_Optimization(Parameters& d,
         return value.to<bool>(d.include.reserve.primary);
     if (key == "include-exportmps")
         return value.to<bool>(d.include.exportMPS);
+    if (key == "include-adequacypatch")
```

Comment:
Check indentation, use the `clang-format` tool

Reply:
Should be correct now